### PR TITLE
Section added for optizmo access endpoint and campaign links addition

### PR DESCRIPTION
**Pull request by diablodrew**
Dredd tests pass!

### CODE REVIEW

**Comment by jaydiablo** [comment]
In the `Campaign Lite` model could you add these two links in there as well?

```
- suppression
    - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/suppression?publisher_id=10000 (string)
- optizmoaccess
    - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/optizmoaccess?publisher_id=10000 (string) ~ If available
```

I noticed that the API has these links (if available) in the Campaign Collection but it isn't documented.

File: apiary.apib [hunk 1]
```diff
@@ -1178,6 +1199,8 @@ There are a lot of filtering capabilities for this report. Most of the columns r
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/creatives_export?publisher_id=10000 (string)
     - suppression
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/suppression?publisher_id=10000 (string)
+    - optizmoaccess
+        - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/optizmoaccess?publisher_id=10000 (string) ~ If available
```
Comment (jaydiablo) [inline apiary.apib]: This `~ If available` doesn't render in Apiary, was this a syntax you found for an optional property?

Comment (diablodrew) [reply apiary.apib]: What looked like a ~ was a -, though I was just trying to add a "comment" there.  I'll see if there's a proper way to flag an optional property

Comment (diablodrew) [reply apiary.apib]: Hmm, nullable might apply here?  

"nullable - instance of this type Value MAY be unset (e.g. null or nil)."

File: apiary.apib [hunk 2]
```diff
@@ -1176,8 +1201,10 @@ There are a lot of filtering capabilities for this report. Most of the columns r
 - `_links` (object)
     - `creatives_export`
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/creatives_export?publisher_id=10000 (string)
-    - suppression
+    - `suppression`
```
Comment (diablodrew) [inline apiary.apib:1204]: Not sure how pertinent this change was, I noticed creatives_export had it above so added to suppression and optizmoaccess.

Comment (jaydiablo) [reply apiary.apib:1204]: I think it's only important if the field name contains characters that might break the parsing, so shouldn't be harmful to add.

File: apiary.apib [hunk 3]
```diff
@@ -1164,8 +1185,12 @@ There are a lot of filtering capabilities for this report. Most of the columns r
 
 ## Campaign Lite (Campaign Lite Base)
 - `_links` (object)
-    - self
+    - `self`
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1?publisher_id=10000 (string)
+    - `suppression`
```
Comment (jaydiablo) [inline apiary.apib:1190]: Technically I think this is nullable too, if a campaign doesn't have a suppression file, this won't be present no?

**Review by jaydiablo — APPROVED** [review]

**Pull request merged**